libc/utils/docgen/unistd.yaml

```diff
@@ -0,0 +1,722 @@
+  _POSIX2_SW_DEV:
+    in-latest-posix: ""
+  _POSIX2_UPE:
+    in-latest-posix: ""
+  _XOPEN_CRYPT:
+    in-latest-posix: ""
+  _XOPEN_ENH_I18N:
+    in-latest-posix: ""
+  _XOPEN_REALTIME:
+    in-latest-posix: ""
+  _XOPEN_REALTIME_THREADS:
+    in-latest-posix: ""
+  _XOPEN_SHM:
+    in-latest-posix: ""
+  _XOPEN_UNIX:
+    in-latest-posix: ""
+  _XOPEN_UUCP:
+    in-latest-posix: ""
+
+  _POSIX_ASYNC_IO:
+    in-latest-posix: ""
+  _POSIX_FALLOC:
+    in-latest-posix: ""
+  _POSIX_PRIO_IO:
+    in-latest-posix: ""
+  _POSIX_SYNC_IO:
+    in-latest-posix: ""
+  _POSIX_TIMESTAMP_RESOLUTION:
+    in-latest-posix: ""
+  _POSIX2_SYMLINKS:
+    in-latest-posix: ""
+
+  "NULL":
+    in-latest-posix: ""
+  O_CLOEXEC:
+    in-latest-posix: ""
+  O_CLOFORK:
+    in-latest-posix: ""
+  F_OK:
+    in-latest-posix: ""
+  R_OK:
+    in-latest-posix: ""
+  W_OK:
+    in-latest-posix: ""
+  X_OK:
+    in-latest-posix: ""
+
+  _CS_PATH:
+    in-latest-posix: ""
+  _CS_POSIX_V8_ILP32_OFF32_CFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_ILP32_OFF32_LDFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_ILP32_OFF32_LIBS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_ILP32_OFFBIG_CFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_ILP32_OFFBIG_LDFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_ILP32_OFFBIG_LIBS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_LP64_OFF64_CFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_LP64_OFF64_LDFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_LP64_OFF64_LIBS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_LPBIG_OFFBIG_CFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_LPBIG_OFFBIG_LDFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_LPBIG_OFFBIG_LIBS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_THREADS_CFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_THREADS_LDFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V8_WIDTH_RESTRICTED_ENVS:
+    in-latest-posix: ""
+  _CS_V8_ENV:
+    in-latest-posix: ""
+
+  _CS_POSIX_V7_ILP32_OFF32_CFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_ILP32_OFF32_LDFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_ILP32_OFF32_LIBS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_ILP32_OFFBIG_CFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_ILP32_OFFBIG_LDFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_ILP32_OFFBIG_LIBS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_LP64_OFF64_CFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_LP64_OFF64_LDFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_LP64_OFF64_LIBS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_LPBIG_OFFBIG_CFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_LPBIG_OFFBIG_LDFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_LPBIG_OFFBIG_LIBS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_THREADS_CFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_THREADS_LDFLAGS:
+    in-latest-posix: ""
+  _CS_POSIX_V7_WIDTH_RESTRICTED_ENVS:
+    in-latest-posix: ""
+  _CS_V7_ENV:
+    in-latest-posix: ""
+
+  SEEK_CUR:
+    in-latest-posix: ""
+  SEEK_END:
+    in-latest-posix: ""
+  SEEK_SET:
+    in-latest-posix: ""
```

Comment:
> The <unistd.h> header shall define SEEK_CUR, SEEK_END, and SEEK_SET as described in [<stdio.h>](https://pubs.opengroup.org/onlinepubs/9799919799/basedefs/stdio.h.html).

This just means that unistd.h includes <stdio.h>. But it does define `SEEK_HOLE` and `SEEK_DATA`.
```suggestion
```